src/catalog/src/kvbackend/client.rs

```diff
@@ -33,18 +35,91 @@ use meta_client::client::MetaClient;
 use moka::future::{Cache, CacheBuilder};
 use snafu::{OptionExt, ResultExt};
 
-use crate::metrics::{METRIC_CATALOG_KV_GET, METRIC_CATALOG_KV_REMOTE_GET};
+use crate::metrics::{
+    METRIC_CATALOG_KV_BATCH_GET, METRIC_CATALOG_KV_GET, METRIC_CATALOG_KV_REMOTE_GET,
+};
+
+const DEFAULT_MAX_CACHE_CAPACITY: u64 = 10000;
```

Comment:
```suggestion
const DEFAULT_CACHE_MAX_CAPACITY: u64 = 10000;
```

Reply:
Keep the same as : https://github.com/GreptimeTeam/greptimedb/pull/3277/files#diff-8c8314e72e06586a9aea0e9e4a2ed8c509312768340be34a6204101483e517e6R73